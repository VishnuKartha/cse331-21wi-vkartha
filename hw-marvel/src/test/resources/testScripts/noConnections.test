#Tries to find paths of a graph loaded from an tsv file with a single connection

# To run this test, the noConnections dataset must exist at
#   hw-marvel/src/main/resources/data/noConnections.tsv


LoadGraph noConnections noConnections.tsv

FindPath noConnections Gandhi Martin_Luther_King
FindPath noConnections Nelson_Mandela Martin_Luther_King
FindPath noConnections Nelson_Mandela Gandhi


