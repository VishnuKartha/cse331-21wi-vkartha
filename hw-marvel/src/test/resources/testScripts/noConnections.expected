#Tries to find paths of a graph loaded from an tsv file with a single connection

# To run this test, the noConnections dataset must exist at
# hw-marvel/src/main/resources/data/noConnections.tsv


loaded graph noConnections

path from Gandhi to Martin Luther King:
no path found
path from Nelson Mandela to Martin Luther King:
no path found
path from Nelson Mandela to Gandhi:
no path found


